src/state_gov__master_sanctions_chart.ts

```diff
@@ -0,0 +1,36 @@
+const pdf_table_extractor = require("pdf-table-extractor");
+const download = require('download-pdf')
+
+
+export async function state_gov__master_sanctions_chart(): Promise<{ [key: string]: any }[]> {
+  let res: { [key: string]: any }[] = [];
+  const URLToDownload = "https://www.state.gov/wp-content/uploads/2022/04/MASTER-Sanctions-chart-4.8.22-1.pdf";
```

Comment:
We shouldn't hardcode this URL! It will change with the next version. Instead, we need to understand what web page contains the link with the most recent URL and get it from there.